pkg/generators/forwarding/fluentd/output_conf_legacy_test.go.rej

```diff
@@ -0,0 +1,18 @@
+--- pkg/generators/forwarding/fluentd/output_conf_legacy_test.go
++++ pkg/generators/forwarding/fluentd/output_conf_legacy_test.go
+@@ -939,15 +878,6 @@ var _ = Describe("Generating fluentd leg
+             </store>
+           </match>
+         </label>
+-        <label @_AUDIT>
+-          <match **>
+-            @type copy
+-            <store>
+-              @type relabel
+-              @label @_LEGACY_SYSLOG
+-            </store>
+-          </match>
+-        </label>
+         <label @_INFRASTRUCTURE>
+           <match **>
+             @type copy
```

Comment:
@k-keiichi-rh please remove this file.
Cc: @jcantrill

Reply:
I am sorry for my fault. I submitted https://github.com/openshift/cluster-logging-operator/pull/1140.